src/lib/server/main_loop.c

```diff
@@ -255,9 +255,8 @@ int main_loop_init(void)
 
 #ifdef HAVE_SYSTEMD_WATCHDOG
 	if (sd_watchdog_interval.tv_sec || sd_watchdog_interval.tv_usec) {
-		struct timeval	now;
-		fr_time_to_timeval(&now, fr_event_list_time(event_list));
-		sd_watchdog_event(event_list, &now, NULL);
+		fr_time_t now = fr_time();
+		sd_watchdog_event(event_list, now, NULL);
```

Comment:
Ah I didn't realise it was an internal function.  Just get rid of the intermediary ``now`` and pass in 0, now isn't event used in sd_watchdog_event.

Reply:
@arr2036 done.